src/Microsoft.AspNet.SignalR.JS/README.md

```diff
@@ -1,6 +1,10 @@
 # ASP.NET SignalR JavaScript Client
 
-This is the client for ASP.NET SignalR. Use this package if you are connecting to an application written in ASP.NET (using System.Web, or OWIN self-host). This package is **not compatible** with ASP.NET **Core** SignalR. If you are connecting to an ASP.NET Core application, use the `@microsoft/signalr` package.
+This is the client for ASP.NET SignalR. Use this package **only** if you are connecting to an application written in ASP.NET (using System.Web, or OWIN self-host).
+
+> Note: This package is **not compatible** with ASP.NET **Core** SignalR and is not the recommended client for building serverless applications with Azure SignalR Service.
```

Comment:
Will this show up like "Note: The file extension .md indicates a Markdown file." does on https://docs.npmjs.com/about-package-readme-files? If so, great!

Reply:
It might be best to put this at the very top, even above the "# ASP.NET SignalR JavaScript Client	# ASP.NET SignalR JavaScript Client" header.